### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by TheFixerDevelopment** [bluetigeresw/NameCorrector#1]
Hey, I do like this plugin, but isn't it outdated? I mean we have MCPE 1.2, and you would need XBox to sign into servers. Does it bypass that, or what's the purpose of this plugin? Since 1.2 has released, this plugin is basically useless, isn't it? I just need to know the strait facts about this plugin, and why it can be useful for ALPHA8 support, considering MCPE 1.2 has released, and that you need xBox to login to servers. Thanks.

**Comment by bluetigeresw** [comment]
Nope, it's not outdated, it will work with Xbox accounts and bypass.

As of right now, it will only not work if you use "NameChanger" to give people access to their old accounts; however, I plan on adding support for that very soon! :)

**Comment by bluetigeresw** [comment]
@TheFixerDevelopment Nope, it's not outdated, it will work with Xbox accounts and bypass.

As of right now, it will only not work if you use "NameChanger" to give people access to their old accounts; however, I plan on adding support for that very soon! :)

**Comment by TheFixerDevelopment** [comment]
So this plugin basically allows you to change your usernames without a need of an XBox account?
